### ISSUE TITLE
NSLogger works with simulator but not actual device

### ISSUE THREAD

**Issue by WadeSellers** [fpillet/NSLogger#187]
Hi... I'm not sure what happened. When I run my app in the Xcode simulator, the NSLogger desktop window pops up and all is good. 

On my phone however, I do not get the same NSLogger desktop popup when I run the app. 

I have verified that my phone is working on the same wifi network that my Mac is so it's not a difference of networks here.

I'm wondering if there's something that might have only been setup in "Debug" and not in "Release" within Xcode. Also, I am integrating this into a Swift app through the use of Bridging Headers.

**Comment by fpillet** [comment]
Are you running 10.11 ? A few other people are hitting the same issue -- it's a tough one to debug, as I can't reproduce it here. You may want to check out #178 for a discussion of the issue and potential solutions.

So far, my best guess is that it's something to do with SSL handshake, gatekeeper and the firewall. But at this stage I unfortunately can't offer a solution (other than establishing a non-SSL connection).

Regarding Swift apps, I am myself using NSLogger in my own Swift apps without trouble. This doesn't seem to be part of the issue you're having.

**Comment by WadeSellers** [comment]
So What I found here is that it must be an internet related thing. When I switched from the office's network to the cell phone's HotSpot and had my Mac connected to that, NSLogger played nicely all around. It must be something to do with the whole SSL thing being discussed or such.  I'm curious now to try and test this internet theory again when I get home this evening and see if NSLogger works at home where my home connection is of typical home network security.
